src/utils/css_styler.tsx

```diff
@@ -1,11 +1,36 @@
-// Utility functions to add CSS to HTML style element
+// Utility functions to concatenate view components CSS
+import {IDictionary} from '../models';
 
-let pageCSS = '';
+// Concatenate each component CSS into this variable
+let componentsCSS = '';
 
-export function addCSSToHead(css: string): void {
-  pageCSS += css;
+// Media queries width in px configuration
+const MEDIA_QUERIES_WIDTH_CONFIG: IDictionary<string> = {
+  xsMin : '320px', // Custom, iPhone Retina
+  smMin : '480px', // Extra Small Devices, Phones
+  mdMin : '768px', // Small Devices, Tablets
+  lgMin : '992px', // Medium Devices, Desktops
+  xlgMin: '1200px' // Large Devices, Wide Screens
 }
 
-export function getAllPageHeadCSS(): string {
-  return pageCSS;
+// Supported media queries options
+interface IMediaQueries {
```

Comment:
No idea why but I don't like the Dictionary approach because it opens up possibility of misspelling the object keys. Have you tried stopping the watcher and starting it again? Try also `npm run watch`. If it continues not working pls let me know.

Have in mind that some times when you make changes to some interfaces, webpack doesn't notice them and it gives you some weird compilation errors. So in those cases you kill the watcher and you run it again.